deploy/stacks/bootstrap_stack.py

```diff
@@ -0,0 +1,201 @@
+from aws_cdk import Duration
+from aws_cdk import (
+    aws_iam as iam,
+    aws_ec2 as ec2,
+    aws_lambda as _lambda,
+)
+from aws_cdk.triggers import TriggerFunction
+
+from .pyNestedStack import pyNestedClass
+
+
+class BootstrapStack(pyNestedClass):
+    def __init__(
+        self,
+        scope,
+        id,
+        handler,
+        envname='dev',
+        resource_prefix='dataall',
+        vpc=None,
+        vpce_connection=None,
+        ecr_repository=None,
+        image_tag=None,
+        pivot_role_name=None,
+        connections: [ec2.Connections] = [],
+        **kwargs,
+    ):
+        super().__init__(scope, id, **kwargs)
+
+        if self.node.try_get_context('image_tag'):
+            image_tag = self.node.try_get_context('image_tag')
+        image_tag = f'lambdas-{image_tag}'
+
+        env = {'envname': envname, 'LOG_LEVEL': 'INFO'}
+
```

Comment:
Good catch and I already tried it, the problem is that this creates a cycling dependency because you correctly noted that CDK already detects the dependency implicitly.